spark/export_avro.py

```diff
@@ -0,0 +1,104 @@
+import json
+from collections import defaultdict
+
+from fastavro import writer
+
+from utils.avro import create_avro_from
+from utils.encoding import *
+
+
+def export_avro(spark, schema, metadata, dd_tables, db_url, db_user, db_pass):
+    node_label, edge_label = dd_tables
+    db = spark.read.format('jdbc'). \
+        options(
+        url=db_url,
+        user=db_user,
+        password=db_pass,
+        driver='org.postgresql.Driver')
+
+    all_tables = db.options(dbtable='information_schema.tables').load().select('table_name')
+
+    it = defaultdict(list)
+
+    edge_tables = [t['table_name'] for t in all_tables.toLocalIterator() if t['table_name'].startswith('edge_')]
+
+    for e, v in edge_label.items():
+        if e in edge_tables:
+            it[v['src']].append(e)
+
+    table_logs = '{:>40} = {:<10} [{:<12}]'
```

Comment:
put some comments to explain or give it a name so that it is more understandable for others